aten/src/ATen/core/op_registration/op_registration_test.cpp

```diff
@@ -237,7 +236,7 @@ TEST(OperatorRegistrationTest, givenOpWithoutKernels_whenRegisteringWithSchema_t
   expectThrows<c10::Error>([&] {
     callOp(*op, dummyTensor(c10::DispatchKey::CPU));
   }, "Could not run '_test::dummy' with arguments from the 'CPU'"
-  " backend. '_test::dummy' is only available for these backends: [].");
```

Comment:
Why did you remove this from the test? If I understand your PR correctly, you're still emitting this part of the error message.